libs/positions/src/lib/positions-table.tsx

```diff
@@ -140,14 +115,33 @@ const ButtonCell = ({
   );
 };
 
-const EmptyCell = () => '';
+const progressBarValueFormatter = ({
```

Comment:
chore (non blocking): Looks a bit like this logic is duplicated in accounts-table.tsx, Seems like the Progressbar component should handle this. Can be a separate PR

Reply:
it is  similar with the one in the positions-table, but the range calculation is different. I moved to ui-toolkit progress bar selectors and cell renderers.